app/main.py

```diff
@@ -129,6 +129,20 @@ def create_presigned_url(expiration=3600):
 
     return response
 
+# Reverse proxy for objects from S3, a simple get object
+# operation. This is used by scaffoldvuer and its 
+# important to keep the relative <path> for accessing
+# other required files.
+@app.route("/object-resource/<path:path>")
+def download_download_url(path):
```

Comment:
I would probably call this simply `download_url`.

Reply:
I meant to put down direct_download_url there... hmmm...